src/test/resources/net/pms/util/prettified_filenames_metadata.json

```diff
@@ -0,0 +1,223 @@
+[
+	{
+		"comment": "Video of a TV episode",
+		"filename": "Universal.Media.Server.S01E02.720p.mkv",
+		"prettified": "Universal Media Server - 102",
+		"metadata": {
+			"type": "tv-series-episodes",
+			"series": "Universal Media Server",
+			"season": 1,
+			"episodes": [2]
```

Comment:
These metadata fields don't seem to be used, are these for a future improvement?

Reply:
Yes, as I noticed that at some other place in the code metadata is guessed from filename to double-check the downloaded info.

Reply:
@stasinos it would be cool to grow this unit test into more like an integration test so it can use this metadata to verify the database data was written correctly - not in this PR but at some point in the future. As our automated testing matures I'd like to see more of that across the board.